### ISSUE TITLE
A long variable can't be fully displayed in renaming prompt box

### ISSUE THREAD

**Issue by UncleBill** [neoclide/coc.nvim#2962]
## Result from CocInfo

```
## versions

vim version: VIM - Vi IMproved 8.2 8022576
node version: v14.15.4
coc.nvim version: 0.0.80-d09f35455b
coc.nvim directory: /Users/unclebill/.vim/plugged/coc.nvim
term: dumb
platform: darwin

## Log of coc.nvim

2021-03-12T11:16:14.009 INFO (pid:6553) [coc-git] - Looking for git in: git
2021-03-12T11:16:14.044 INFO (pid:6553) [services] - registered service "highlight"
2021-03-12T11:16:14.211 INFO (pid:6553) [language-client-index] - highlight started with 6555
2021-03-12T11:16:14.256 INFO (pid:6553) [plugin] - coc.nvim 0.0.80-d09f35455b initialized with node: v14.15.4 after 377ms
2021-03-12T11:16:14.721 INFO (pid:6553) [attach] - receive notification: doAutocmd [ 1 ]
2021-03-12T11:16:18.876 INFO (pid:6553) [language-client-index] - eslint started with 6557
2021-03-12T11:16:18.925 INFO (pid:6553) [services] - registered service "tsserver"
2021-03-12T11:16:20.585 INFO (pid:6553) [attach] - receive notification: rename []
2021-03-12T11:16:20.618 INFO (pid:6553) [attach] - receive notification: doAutocmd [ 1 ]
2021-03-12T11:16:46.519 INFO (pid:6553) [attach] - receive notification: doAutocmd [ 1 ]
2021-03-12T11:16:46.524 INFO (pid:6553) [attach] - receive notification: doAutocmd [ 1 ]
2021-03-12T11:18:34.177 INFO (pid:6553) [attach] - receive notification: showInfo []

```

## Describe the bug

See the screenshot below, the `VERY_LONG_LONG_LONG_LONG_VARIABLE_NAME` should be fully displayed.

## Reproduce the bug

**We will close your issue when you don't provide minimal vimrc and we can't
reproduce it**

- Create file `mini.vim` with：

  ```vim
  set nocompatible
  set runtimepath^=/path/to/coc.nvim
  filetype plugin indent on
  syntax on
  set hidden
  ```

- Start (neo)vim with command: `vim -u mini.vim`

- Operate vim.

## Screenshots (optional)

![image](https://user-images.githubusercontent.com/1141198/110887174-cc06ef80-8324-11eb-9cc4-44b468a14e0c.png)

### Moving cursor backward

Heading texts not show up when move cursor backward

![image](https://user-images.githubusercontent.com/1141198/110889406-08d4e580-8329-11eb-9ee5-89776fdb978e.png)

**Comment by chemzqm** [comment]
Fixed by 02e0b81a

**Comment by chemzqm** [comment]
Fixed by 02e0b81a

**Comment by UncleBill** [comment]
Cool! Thanks!
